storage/innobase/xtrabackup/test/t/xb_stats.sh

```diff
@@ -17,15 +17,25 @@ xtrabackup --datadir=$mysql_datadir --prepare --apply-log-only --target-dir=$top
 vlog "===> xtrabackup --stats --datadir=$topdir/backup"
 run_cmd_expect_failure $XB_BIN $XB_ARGS --stats --datadir=$topdir/backup
 
-if ! grep -q "Cannot find log file" $OUTFILE
+if ! grep -q "Cannot create redo log files" $OUTFILE
 then
     die "Cannot find the expected error message from xtrabackup --stats"
 fi
 
-# Now create the log files in the backup and try xtrabackup --stats again
-
 xtrabackup --datadir=$mysql_datadir --prepare --target-dir=$topdir/backup
 
-xtrabackup --stats --datadir=$topdir/backup
+vlog "Now start the server so it creates redo log files"
+stop_server
+rm -rf $mysql_datadir
+xtrabackup --copy-back --datadir=$mysql_datadir --target-dir=$topdir/backup
+
+start_server
+innodb_wait_for_flush_all
+mysql -e "set global innodb_fast_shutdown=0"
+shutdown_server
+
+vlog " checking Stats on data directory "
+
+xtrabackup --stats --datadir=$mysql_datadir
```

Comment:
This was not the original intention of the test. It looks like stats should be allowed on prepared datadir. I think you should ad run_cmd_expect failure with xtrabckup cribbing about missing redo log files on the prepared dir.

we will add it as limitation

Reply:
We have that in xb_stats.sh
`if ! grep -q "Cannot create redo log files" $OUTFILE
then
    die "Cannot find the expected error message from xtrabackup --stats"
fi`